sdk/src/test/java/com/silanis/esl/sdk/builder/SigningLogoBuilderTest.java

```diff
@@ -0,0 +1,24 @@
+package com.silanis.esl.sdk.builder;
+
+import com.silanis.esl.sdk.SigningLogo;
+import org.junit.Test;
+
+import java.util.Locale;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class SigningLogoBuilderTest {
+    @Test
+    public void withSpecifiedValues() {
+        String logoString = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEUAAABnCA";
+        SigningLogoBuilder signingLogoBuilder = SigningLogoBuilder.newSigningLogo()
+                .withLanguage(new Locale("en"))
+                .withImage(logoString);
+
+        SigningLogo result = signingLogoBuilder.build();
+
+        assertThat( "language was not set correctly", result.getLanguage(), is(Locale.ENGLISH ));
+        assertThat( "image was not set correctly", result.getImage(), is(logoString ) );
+    }
+}
```

Comment:
This part has already been modified and doesn't need to be modified. Please rebase the dev branch.